### ISSUE TITLE
[CS 1.6] [Feature Request] ex_interp minimum requirement by server

### ISSUE THREAD

**Issue by Maxi605** [ValveSoftware/halflife#3122]
I would love to have a server-side cvar to set a minimum value for client's ex_interp since this grants a huge advantage in certain map/mods (Mostly with knives)

**Comment by fox3562** [comment]
Indeed, why should we see the "real" (more accurate) position of the model (at 0.01 ), when the client (at 0.1) draws the predicted position... (sarcasm) With the same success, you can limit all the variables on the client. In most cases, a certain advantage is given only by the brain, which knows which variable is responsible for what.

**Comment by Maxi605** [comment]
> Indeed, why should we see the "real" (more accurate) position of the model (at 0.01 ), when the client (at 0.1) draws the predicted position... (sarcasm) With the same success, you can limit all the variables on the client. In most cases, a certain advantage is given only by the brain, which knows which variable is responsible for what.
> 2kisak-valve: do you really agree with this? O_o

Well, the suggestion is about a SERVER-SIDE VAR, that way people are giving the choice of limiting or not the usage of the client var on their own servers.

**Comment by RauliTop** [comment]
What are the advantages? Can you show in a video?

I tried with knife distance, but for me, it's the same.
with ex_interp 0.1 and ex_interp 0.01

**Comment by fox3562** [comment]
the [WH Blocker](http://www.dedicated-server.ru/vbb/showthread.php?t=25639) metaplagin has a parameter that allows you to limit ex_interp on the server side `(interp 0.01 / 0.1 is the minimum ex_interp threshold for the client)`

**Comment by Maxi605** [comment]
> the [WH Blocker](http://www.dedicated-server.ru/vbb/showthread.php?t=25639) metaplagin has a parameter that allows you to limit ex_interp on the server side
> `(interp 0.01 / 0.1 is the minimum ex_interp threshold for the client)`
> P.s. do you have any idea what ex_interp is, what determines parameter and how it works?

That's quite interesting. Thanks!

The determinate parameter is 0.1, it should be brought down to 0.01 or add a minimum requirement to prevent people from getting an advantage over others just because they aren't aware, but people would be mad about it, that's why i suggested as a server-side cvar
